src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -330,6 +330,21 @@ public void timeEvent(final String eventName) {
     }
 
     /**
+     * Map version of the track API below.
```

Comment:
Let's make this comment a stand-alone message, with an `{@link }` to the JSON version of the API.